bioy_pkg/subcommands/classifier.py

```diff
@@ -274,7 +275,12 @@ def condense_ids(df, tax_dict, ranks, max_group_size):
         condensed.items(),
         columns=[ASSIGNMENT_TAX_ID, 'condensed_id'])
     condensed = condensed.set_index(ASSIGNMENT_TAX_ID)
-    assignment_hash = hash(frozenset(condensed.condensed_id.unique()))
+
+    if threshold_assignments:
```

Comment:
A comment here would be helpful in explaining what's going on.  It's not clear to me without putting my hands on the code what exactly hash(frozenset()) is doing (I know that downstream blast hits are grouped by assignment_hash).  In particular, in both cases, which blast hits get mapped to the same hash values?